lib/topology/platforms/utils.py

```diff
@@ -94,9 +94,11 @@ def load_nodes(self, cache=True):
 
             try:
                 node = ep.load()
-            except:
+            except Exception as error:
                 log.error(
```

Comment:
If I understand the docs correctly exception would log the error message ( 'Unable to load node from plugin') and a traceback, currently it only logs  the error message.

 Given that this particular error occurs when a plugin fails on load for some reason I would argue that people do want the trace as it currently is the frameworks just informs that a plugin failed to load, and not why. Logging the trace is minimal spam since it wont show unless something fails to load and generally if a plugin fails to load no test can run, so having extra information is useful when trying to decipher why the test won't run. 

I recently had to debug a plugin load error and I ended up essentially using pdb to print the exception and would be nice and a little bit easier to have this info on the logs directly.